release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -523,13 +523,13 @@ For more information, see xref:../security/tls-security-profiles.adoc#tls-profil
 [id="ocp-release-notes-web-console_{context}"]
 === Web console
 
-[id="ocp-administrator-perspective_{context}"]
-==== Administrator perspective
+Starting with {product-title} 4.19, the perspectives in the web console have unified to simplify navigation, reduce context switching, streamline tasks, and provide users with a more cohesive {product-title} experience.
 
-[id="ocp-developer-perspective_{context}"]
-==== Developer Perspective
+With this unified design, there is no longer a *Developer* perspective by default; however,
+_all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission to certain features from the cluster owner. You can still enable the *Developer* perspective.
+//See <module> for more information on enabling the *Developer* perspective. A quick start is also available with more information.
 
-This release introduces the following updates to the *Developer* perspective of the web console:
+The *Getting Started* pane in the web console also provides resources such as, a tour of the console, information on setting up your cluster, and links to explore new features and capabilities.
```

Comment:
We have also added quick start to `Enable the developer perspective` and added it to `Getting Started` pane. Can we also add it here?